serve/tests/unittest/test_engine_with_samplers.py

```diff
@@ -422,6 +547,8 @@ def _test_json_mode(
     _test_stop(sync_engine)
     _test_logprobs(sync_engine)
     _test_logprobs_mixed_requests(sync_engine)
+    _test_num_sequences(sync_engine)
+    _test_logit_bias(sync_engine)
```

Comment:
This tests seems to always fail when I run all tests in `test_engine_with_samplers.py`. But if I comment out other tests and run only this one, it works. So there is a strange issue in the tests. Can you take a look and send a fix to the ollm repo?